pkg/tests/api/shorturl/short_url_test.go

```diff
@@ -31,10 +33,12 @@ func TestMain(m *testing.M) {
 func TestShortURL(t *testing.T) {
 	dir, path := testinfra.CreateGrafDir(t, testinfra.GrafanaOpts{
 		AppModeProduction: true,
+		DisableAnonymous:  true,
```

Comment:
Tests were running without using the authentication, I've disabled anonymous and added unauthenticated cases.